feathr_project/feathr/_feature_registry.py

```diff
@@ -807,3 +807,61 @@ def get_features_from_registry(self, project_name: str, workspace_path: str):
         with open(feature_gen_file, "w") as online_config:
             online_config.write(feature_gen_conf_content)
         logger.info("Writing online configuration from feathr registry to {}", feature_gen_file)
+   
+    def get_feature_by_fqdn_type(self, qualifiedName, typeName):
```

Comment:
What's the use case of this function? Can it be merged with "get_feature_by_fqdn()"

Reply:
In some cases, you can do a more optimized search if you know the name and type, in case of just FQDN we need to do a plain search and look for results.fqdn == given FQDN.

Hence keeping this method there, the UI would be able to use it since they would most likely know what type of feature user wants to see.